src/main/java/unicash/ui/SummaryWindow.java

```diff
@@ -17,26 +23,32 @@
  */
 public class SummaryWindow extends UiPart<Stage> {
 
-    public static final String USER_GUIDE_URL = "https://ay2324s1-cs2103-t16-3.github.io/tp/";
-    public static final String SUMMARY_MESSAGE = "Here is your summary:";
+    public static final String SUMMARY_MESSAGE = "Expense Summary:";
+    public static final String SUMMARY_MESSAGE_WHEN_NO_TRANSACTIONS = "There are no transactions available";
```

Comment:
Will this ever be a case that occurs? Since we're intentionally restricting users from reaching the summary  window if they don't have any transactions right?

Reply:
This might occur if the user opens the summary window before clearing all expenses from UniCa$h. In such a scenario, I designed `SummaryWindow` such that after the user clears all expenses (i.e. there is no more data to plot), the title of the window changes from "Expense Summary:" to "There are no transactions available". In hindsight, I should change it to "There are no expenses available" instead.

Here is what the summary window looks like now if the situation described above occurs:

![image](https://github.com/AY2324S1-CS2103-T16-3/tp/assets/67943858/27605433-f83c-4268-a88e-8acd874da2a7)

Reply:
I explained the reason why I designed it this way in the PR description! Let me know what you think

"However, if the summary window was already opened by the user beforehand, and they cleared all transactions, I intentionally designed the summary window NOT to close by itself. Otherwise, the disappearing window might confuse the user."

Reply:
> if the user opens the summary window before clearing all expenses from UniCa$h.

So you're saying that the user runs the clear_transactions command after they run summary?